src/index.js

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import * as ReactDOMClient from 'react-dom/client';
+import { Header } from "./Header/Header";
```

Comment:
Try to use single quotes, instead of double quotes, wherever you can and keep it consistent.